app/controllers/user_credentials_controller.rb

```diff
@@ -25,7 +25,7 @@ def import
       if new_credentials.map(&:save).all?
         format.html { redirect_to user_credentials_path, notice: "The file was successfully uploaded."}
       else
-        format.html { redirect_to user_credentials_path, notice: 'One or more of the uploaded was not imported due to duplicate credential name error', status: 422 }
+        format.html { redirect_to user_credentials_path, notice: 'One or more of the uploaded was not imported due to duplicate credential name error'}
```

Comment:
I decided to remove the `status: 422` symbol because according to the doc the redirection will fail. It can redirect properly only if its a `3xx error`.